### PR TITLE
Add Bsky for Yani

### PR DESCRIPTION
**Pull request by cosimameyer**
Hi @yabellini,  

I'm working on adding a Bluesky integration for the R-Ladies bot and saw that you're also on Bluesky. Please let me know 

- if you agree to add your handle here as well so that the bot can use it to tag you when posting about your posts
- if you're also OK with being featured in an R-Ladies starter pack (I assume yes, but double-checking with everyone :))

Thank you ☺️

### CODE REVIEW

File: blogs/yabellini.netlify.app.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
       "social_media": [{
          "twitter": "yabellini",
```
Comment (yabellini) [inline blogs/yabellini.netlify.app.json:12]:
```suggestion

```

File: blogs/yabellini.netlify.app.json [hunk 2]
```diff
@@ -11,6 +11,7 @@
...
          "mastodon": "@yabellini@fosstodon.org",
+         "bluesky": "@yabellini.bsky.social",
          "github": "yabellini",
          "instagram": "yabellini",
```
Comment (yabellini) [inline blogs/yabellini.netlify.app.json:16]:
```suggestion
         "instagram": "yanina.bellini",
```

File: blogs/yabellini.netlify.app.json [hunk 3]
```diff
@@ -11,6 +11,7 @@
...
          "youtube": "channel/UCA7IUNba2IjQfrVVjh2Whmw",
```
Comment (yabellini) [inline blogs/yabellini.netlify.app.json:17]:
```suggestion
         "youtube": "@yabellini",
```

**Comment by yabellini** [comment]
@cosimameyer thank you so much!! I update some of my social media handles.  I don't exactly know how the YouTube one works. But it has my name.  
Also, I think I already am in the R-Ladies Starter Kit for BlueSky, but of course I agree on being on the Kit and your awesome bots tag me :-) 
Let me know if you need anything else from me!

**Comment by cosimameyer** [comment]
Thank you so much! By the way, would you be interested (in general) to also have your YouTube feed (or R-specific channel) being shared with the bot? It's a feature that I use more extensively with the PyLadies bot but I think it would be amazing for the R-Ladies bot (and R-Ladies chapters :)) as well :)

**Comment by cosimameyer** [comment]
--- 

Just as a ref for #97

**Pull request merged**